gen/schema/schema.yaml

```diff
@@ -7,7 +7,7 @@ put_create: bool(required=False) # Set to true if the PUT request is used for cr
 no_update: bool(required=False) # Set to true if the PUT request is not supported
 no_delete: bool(required=False) # Set to true if the DELETE request is not supported
 data_source_name_query: bool(required=False) # Set to true if the data source supports name queries
-minimum_version: str(required=False) # Define a minimum supported version
```

Comment:
I would keep the `minimum_version` attribute as-is, which is meant to be used if the corresponding API endpoint(s) have only been introduced with a specific version. There is also a documentation helper function (https://github.com/netascode/terraform-provider-fmc/blob/0f49e47587176b4c7f2b1e481a3e36f9699dd1e6/internal/provider/helpers/description_builder.go#L33) to update the documentation accordingly.

Instead of making this a map, I would suggest to simply introduce more specific attributes such as `minimum_version_object_support` and `minimum_version_bulk_delete`.

Reply:
Reverted changes, following the suggestion